pipestat/html_reports_pipestat.py

```diff
@@ -577,20 +577,28 @@ def get_nonhighlighted_results(self, types):
             are not highlighted
         """
         results = []
-        for k, v in self.schema["samples"].items():
-            if self.schema["samples"][k]["type"] in types:
-                if "highlight" not in self.schema["samples"][k].keys():
-                    results.append(k)
-                # intentionally "== False" to exclude "falsy" values
-                elif self.schema["samples"][k]["highlight"] is False:
-                    results.append(k)
-        for k, v in self.schema["project"].items():
-            if self.schema["project"][k]["type"] in types:
-                if "highlight" not in self.schema["project"][k].keys():
-                    results.append(k)
-                # intentionally "== False" to exclude "falsy" values
-                elif self.schema["project"][k]["highlight"] is False:
-                    results.append(k)
+        try:
+            for k, v in self.schema["samples"].items():
+                if self.schema["samples"][k]["type"] in types:
+                    if "highlight" not in self.schema["samples"][k].keys():
+                        results.append(k)
+                    # intentionally "== False" to exclude "falsy" values
+                    elif self.schema["samples"][k]["highlight"] is False:
+                        results.append(k)
+        except KeyError:
+            pass
+
+        try:
+            for k, v in self.schema["project"].items():
+                if self.schema["project"][k]["type"] in types:
+                    if "highlight" not in self.schema["project"][k].keys():
+                        results.append(k)
+                    # intentionally "== False" to exclude "falsy" values
+                    elif self.schema["project"][k]["highlight"] is False:
+                        results.append(k)
+        except KeyError:
+            pass
```

Comment:
and here too, I'm just worried when I see "except: pass" -- maybe just a comment as to why this is needed would satisfy.